lib/services/panduit-obm-service.js

```diff
@@ -143,14 +150,15 @@ function panduitObmServiceFactory(
         });
     };
 
-    PanduitObmService.prototype._runInternal = function (command, host, community, file) {
+    PanduitObmService.prototype._runInternal = function (command, host, community, file, retries) {
         return this.run({
-            command: file || 'snmpset',
+            command: file,
             args: [
                 '-v2c',
                 '-c', community,
                 host
-            ].concat(command)
+            ].concat(command),
+            retries: retries
```

Comment:
this.run() command in base-obm-service always expect "retries" as its parameter. If it is not specified, run() in child-process.js of on-core has assigned a 0 as its default value.